rare/components/tabs/games/game_info/game_info.py

```diff
@@ -119,52 +125,65 @@ def verify(self):
                     self.igame.title
                 ),
             )
             return
         self.verify_widget.setCurrentIndex(1)
         verify_worker = VerifyWorker(self.game.app_name)
-        verify_worker.signals.status.connect(self.verify_staistics)
-        verify_worker.signals.summary.connect(self.finish_verify)
+        verify_worker.signals.started.connect(self.signals.progress.started)
+        verify_worker.signals.status.connect(
+            lambda num, total, app: self.signals.progress.value.emit(app, num * 100 // total)
+        )
+        verify_worker.signals.status.connect(self.verify_status)
+        verify_worker.signals.summary.connect(
+            lambda fail, miss, app: self.signals.progress.finished.emit(app, not fail)
+        )
+        verify_worker.signals.summary.connect(self.verify_summary)
         self.verify_progress.setValue(0)
         self.verify_threads[self.game.app_name] = verify_worker
         self.verify_pool.start(verify_worker)
 
-    def verify_staistics(self, num, total, app_name):
+    def verify_status(self, num, total, app_name):
         # checked, max, app_name
         if app_name == self.game.app_name:
+            self.signals.progress.value.emit(app_name, num * 100 // total)
             self.verify_progress.setValue(num * 100 // total)
 
-    def finish_verify(self, failed, missing, app_name):
+    def verify_summary(self, failed, missing, app_name):
         if failed == missing == 0:
             QMessageBox.information(
                 self,
-                "Summary",
-                "Game was verified successfully. No missing or corrupt files found",
+                self.tr("Summary - {}").format(self.igame.title),
+                self.tr("Game was verified successfully. No missing or corrupt files found"),
             )
             igame = self.core.get_installed_game(app_name)
             if igame.needs_verification:
                 igame.needs_verification = False
                 self.core.lgd.set_installed_game(self.igame.app_name, igame)
-                self.verification_finished.emit(igame)
+                self.signals.game.verified.emit(self.igame.app_name)
         elif failed == missing == -1:
-            QMessageBox.warning(self, "Warning", self.tr("Something went wrong"))
-
+            QMessageBox.warning(
+                self,
+                self.tr("Warning - {}").format(self.igame.title),
+                self.tr("Something went wrong")
+            )
         else:
             ans = QMessageBox.question(
                 self,
-                "Summary",
+                self.tr("Summary - {}").format(self.igame.title),
                 self.tr(
-                    "Verification failed, {} file(s) corrupted, {} file(s) are missing. Do you want to repair them?"
+                    "Verification failed, "
+                    "{} file(s) corrupted, {} file(s) are missing."
+                    "Do you want to repair them?"
```

Comment:
space

...no, joking. This breaks translation yet again, there should be a space at the end of line 179.

Reply:
Good catch